string asset_id
geometry_msgs/Pose pose
---
bool status
